config.yaml

```diff
@@ -289,23 +289,24 @@ postgresql_client:
   # Default values for the Postgres DB name, host, username and password
   # For external postgres installation, update accordingly
   host: localhost
-
-  # Password to use when connecting to the database with the postgres user
+  # Azure DBaaS instance's domain which will result in <username>@<azure_domain>
+  # when Cloudify attempts to connect to the DB.
+  # e.g:
+  #  azure_domain: azure_dbaas_example
+  #  server_username: azure_dbaas_username
+  #  result - azure_dbaas_username@azure_dbaas_example
+  azure_domain: ''
+  # Server user name, password, and DB to use when connecting to the database
+  # for Cloudify db initialization and population, postgres user and db are
+  # used by default.
   # This is only relevant for external postgres installations when you enable
   # remote connections
   # THE PASSWORD WILL BE REMOVED FROM THE FILE AFTER THE INSTALLATION FINISHES
-  postgres_password: ''
-
-  # SSL must be enabled for external databases - provide proper certificates
-  # This setting will be ignored (treated as true) if an external DB or
-  # cluster is used
-  ssl_enabled: false
-  # If true, client SSL certificates will need to be supplied and the Postgres
-  # server will verify each client connection
-  ssl_client_verification: false
+  server_db_name: postgres
```

Comment:
I think it will be more clear to switch the prefix to 'postgres' instead 'server'

Reply:
`postgres` has several meanings here: user, db, server implementation
Until now we used to always connect to the `postgres` DB since we connected with the `postgres` user (without supplying a DB it automatically take the DB name as the user trying to connect), since now we allow a connection with a different user than `postgres`, it's not always that the user trying to connect has a DB of its own.
So, we allow connecting to a different default DB, but as the default, we give it `postgres` (which will probably be used for most cases)

Due to that, I think `server` fits better here then `postgres` since it could mean something different.